### PR TITLE
iOS receipt update

### PR DESCRIPTION
**Pull request by jrouault**
We now use on iOS the global app receipt available from iOS 7. Therefore, theses changes remove compatibility with iOS versions previous 7.

poke @ronkorving

### CODE REVIEW

**Comment by ronkorving** [comment]
Also poke @aogilvie

File: platforms/ios/HelloCordova/Plugins/wizPurchasePlugin/WizPurchasePlugin.m [hunk 1]
```diff
@@ -278,10 +401,9 @@ - (void)paymentQueueRestoreCompletedTransactionsFinished:(SKPaymentQueue *)queue
         NSMutableArray *receipts = [[NSMutableArray alloc] init];
         if ([[[SKPaymentQueue defaultQueue] transactions] count] > 0) {
             for (SKPaymentTransaction *transaction in [[SKPaymentQueue defaultQueue] transactions]) {
-                NSString *receipt = [[NSString alloc] initWithData:[transaction transactionReceipt] encoding:NSUTF8StringEncoding];
                 NSDictionary *result = @{
                      @"platform": @"ios",
-                     @"receipt": receipt,
+                     @"orderId": transaction.transactionIdentifier,
```
Comment (aogilvie) [inline platforms/ios/HelloCordova/Plugins/wizPurchasePlugin/WizPurchasePlugin.m]: yay matches Android

**Comment by aogilvie** [comment]
Code wise LGTM. Not been able to fully test. Christ, they don't make it easy to get the receipts do they!!!

**Comment by jrouault** [comment]
Thanks for the review! I want to make additional changes linked to this PR so I will let it sit for a week and take care of it then.

**Comment by jrouault** [comment]
Updated but I will not merge it yet. In fact, theses changes would utterly break ```getPending``` behaviour: transactions are currently automatically finished and thus the receipt would no longer contain consumables (and non-renewing subscriptions) at next app launch.
More info here: https://developer.apple.com/library/mac/documentation/NetworkingInternet/Conceptual/StoreKitGuide/Chapters/DeliverProduct.html#//apple_ref/doc/uid/TP40008267-CH5-SW5
Solution off the top of my head: remove entirely ```getPending``` and have an API to explicitly finish transactions.

**Pull request closed without merging**